Suggestion for reviewers (#19)
tobyhodges/lesson-sprint-recommendations: during.md

  is merged into the lesson as soon as possible.
  Opening draft pull requests is particularly important at the end of a sprint,
  to capture progress up to that point even if a task hasn't yet been completed.
- **Plan who will review Pull Requests.**

>   Make a plan for who will review each Rull Request.

Reviewer: ```suggestion
  Make a plan for who will review each Pull Request.
```

>   As a group you can create list of reviewers by subject so the person who opens the Pull Reqest knows who to is interested and capable of reviewing.

Reviewer: ```suggestion
  As a group you can create list of reviewers by subject so the person who opens the Pull Request knows who is interested and capable of reviewing and can request a review accordingly.
```

>   Alternatively, maybe you decide on circle of reviewers. E.g. Aaliyah reviews Juan's PRs, Juan reivews Alex's PRs, and Alex reviews Aaliyah's PRs.

Reviewer: ```suggestion
  Alternatively, you may decide on a circle of reviewers. E.g. Aaliyah reviews Juan's PRs, Juan reviews Alex's PRs, and Alex reviews Aaliyah's PRs.
```
